src/Stryker.Core/Stryker.Core/Reporters/MarkdownSummaryReporter.cs

```diff
@@ -0,0 +1,133 @@
+using static FSharp.Compiler.AbstractIL.IL.ILExceptionClause;
+
+namespace Stryker.Core.Reporters
+{
+    /// <summary>
+    /// Markdown result table reporter.
+    /// </summary>
+    public class MarkdownSummaryReporter : IReporter
+    {
+        private readonly StrykerOptions _options;
+        private readonly IAnsiConsole _console;
+        private readonly IFileSystem _fileSystem;
+
+        public MarkdownSummaryReporter(StrykerOptions strykerOptions, IFileSystem fileSystem = null, IAnsiConsole console = null)
+        {
+            _options = strykerOptions;
+            _console = console ?? AnsiConsole.Console;
+            _fileSystem = fileSystem ?? new FileSystem();
+        }
+
+        public void OnStartMutantTestRun(IEnumerable<IReadOnlyMutant> mutantsToBeTested)
+        {
+            // This reporter does not report during the testrun
+        }
+
+        public void OnMutantsCreated(IReadOnlyProjectComponent reportComponent)
+        {
+            // This reporter does not report during the testrun
+        }
+
+        public void OnMutantTested(IReadOnlyMutant result)
+        {
+            // This reporter does not report during the testrun
+        }
+
+        public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
+        {
+            var files = reportComponent.GetAllFiles();
+            if (files.Any())
+            {
+                var filename = _options.ReportFileName + ".md";
+                var reportPath = Path.Combine(_options.ReportPath, filename);
+                var reportUri = "file://" + reportPath.Replace("\\", "/");
+
+                GenerateMarkdownReport(_options, reportPath, files, reportComponent.GetMutationScore());
+
+                _console.WriteLine();
+                _console.MarkupLine("[Green]Your Markdown summary has been generated at:[/]");
+
+                if (_console.Profile.Capabilities.Links)
+                {
+                    // We must print the report path as the link text because on some terminals links might be supported but not actually clickable: https://github.com/spectreconsole/spectre.console/issues/764
+                    _console.MarkupLineInterpolated($"[Green][link={reportUri}]{reportPath}[/][/]");
+                }
+                else
+                {
+                    _console.MarkupLineInterpolated($"[Green]{reportUri}[/]");
+                }
+            }
+        }
+
+        private void GenerateMarkdownReport(StrykerOptions options, string reportPath, IEnumerable<IFileLeaf> files, double mutationScore)
+        {
+            if (!files.Any())
+            {
+                return;
+            }
+
+            var mdSummaryDocument = new MdDocument();
+            mdSummaryDocument.Root.Add(new MdHeading(1, "Mutation Testing Summary"));
+
+            var mdSummary = new MdTable(new MdTableRow(new[] { "File", "% score", "# killed", "# timeout", "# survived", "# no cov", "# error" }));
```

Comment:
I think it makes the most sense to use the same columns as the html report with the exception of the runtime errors:

![image](https://user-images.githubusercontent.com/3595932/185599779-c1efb74c-8f21-4254-840a-f7f41c17a032.png)

The cleartext reporter should probably be updated to align with this instead of the other way around.

Reply:
Yeah - I hadn't read your comment completely, I was struggling to find a metric for runtime errors :D  
How does the Javascript work that metric out?  The code is pretty hard to read...

Reply:
Stryker-net does not use runtime errors, so you should omit it. Sorry if that wasn't clear.

Reply:
No worries at all - you told me and it failed to read it properly.  😂. I’ll update the docs with the new format shortly

Reply:
Docs updated - what do you think?  I certainly agree it look better.

Reply:
Awesome!